torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -229,6 +229,27 @@ def affine_bounding_box(
     ).view(original_shape)
 
 
+def affine_segmentation_mask(
+    img: torch.Tensor,
+    angle: float,
+    translate: List[float],
+    scale: float,
+    shear: List[float],
+    center: Optional[List[float]] = None,
+) -> torch.Tensor:
+    angle, translate, shear, center = _affine_parse_args(angle, translate, scale, shear, center=center)
+
+    center_f = [0.0, 0.0]
+    if center is not None:
+        _, height, width = get_dimensions_image_tensor(img)
+        # Center values should be in pixel coordinates but translated such that (0, 0) corresponds to image center.
+        center_f = [1.0 * (c - s * 0.5) for c, s in zip(center, [width, height])]
+
+    translate_f = [1.0 * t for t in translate]
+    matrix = _get_inverse_affine_matrix(center_f, angle, translate_f, scale, shear)
+    return _FT.affine(img, matrix, interpolation=InterpolationMode.NEAREST.value, fill=None)
```

Comment:
Maybe I'm missing something here, but this seems copy-pasted from `affine_image_tensor`. Why not call that directly?